# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../latex2html/dont_rebuild_ls-R.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./config/install.pl.orig	2005-03-03 10:53:59.813725032 -0300
+++ ./config/install.pl	2005-03-03 10:54:23.658100136 -0300
@@ -394,10 +394,6 @@
         }
       }
       closedir(DIR);
-      if($ok && $cfg{MKTEXLSR}) {
-        print "Info: Running $cfg{MKTEXLSR} to rebuild ls-R database...\n";
-        system($cfg{MKTEXLSR});
-      }
     }
   }
 }
